proxy/src/coco/peer.rs

```diff
@@ -113,6 +125,67 @@ pub fn list_projects(peer: &PeerApi) -> Result<Vec<Project>, error::Error> {
         .collect()
 }
 
+/// Get all peer IDs and their branches.
+///
+/// # Errors
+///
+///   * [`error::Error::LibradLock`]
+///   * [`error::Error::Git`]
+pub fn remotes(
+    peer: &PeerApi,
+    owner: &User,
+    project_urn: &RadUrn,
+) -> Result<Vec<Remote>, error::Error> {
+    let project = get_project(peer, project_urn)?;
+    let peer_id = peer.peer_id();
+    let storage = peer.storage();
+    let repo = storage.open_repo(project.urn())?;
+    let refs = repo.rad_refs()?;
+
+    let (local_branches, local_tags) = with_browser(peer, &project.urn(), |browser| {
+        Ok((source::local_branches(browser)?, source::tags(browser)?))
+    })?;
+
+    let owner = owner.to_data().build()?; // TODO(finto): Dirty hack to make our Verified User into a Draft one
+    let mut remotes = vec![Remote {
+        user: owner,
+        branches: local_branches,
+        tags: local_tags,
+    }];
+
+    for remote in refs.remotes.flatten() {
+        let refs = if remote == peer_id {
+            // TODO(finto): Bug in librad
```

Comment:
This should be fixed.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/636/commits/712efe9e771c199f964b90160ace91ef5bd6dd68